packages/ripple-nuxt-tide/lib/config/defaults.js

```diff
@@ -14,7 +14,7 @@ const defaults = {
   dynamicComponents: [], // Dynamic components importing for Tide mapping.
   middleware: [],
   markupPlugins: [],
-  proxyTimeout: 60000,
+  proxyTimeout: 30000,
```

Comment:
30s is enough for Tide API now. Keep is as low as possible because Proxy server will keep wait Tide API to response even axios has closed the connection(after 10s timeout setting).